multi-agent/tests/regression/test_adapters_import_regression.py

```
@@ -0,0 +1,71 @@
+        os.symlink(mas_dir, os.path.join(tmpdir, "mas"))
+        
+        # Prepare environment for the subprocess
+        env = os.environ.copy()
+        env["PYTHONPATH"] = tmpdir
+        
+        # Act: Run a subprocess to import bootstrap.container
+        # We run from a neutral directory (/tmp) to ensure the current directory
+        # (which contains 'adapters') is not implicitly added to sys.path.
+        result = subprocess.run(
+            [sys.executable, "-c", "import bootstrap.container"],
+            cwd="/tmp",
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)